spec/omniauth/auth_hash_spec.rb

```diff
@@ -8,6 +8,11 @@
     expect(subject.info.first_name).to eq('Awesome')
   end
 
+  it 'does not try to parse `string` as InfoHash' do
```

Comment:
`undefined method #each_pair on String` occurs when you try to initialize `Hashie::Mash` with a string.
And it's not tested. Your test describes that it simply parses `info` field which is assigned with a hash as `InfoHash` object. If you assign a string to `subject.info` if would fail. Also `omniauth` tries to parse every `info` on any level of nesting and it causes some troubles, but it's not covered here.